src/integrationTest/java/com/mongodb/spark/sql/connector/read/partitioner/AutoBucketPartitionerTest.java

```diff
@@ -49,11 +49,7 @@ public class AutoBucketPartitionerTest extends PartitionerTestCase {
 
   @Override
   List<String> defaultReadConfigOptions() {
-    return asList(
-        ReadConfig.PARTITIONER_OPTIONS_PREFIX + PARTITION_CHUNK_SIZE_MB_CONFIG,
-        "1",
-        ReadConfig.PARTITIONER_OPTIONS_PREFIX + SamplePartitioner.SAMPLES_PER_PARTITION_CONFIG,
```

Comment:
No need for the sample partitioner configuration.

Reply:
No plans to remove at this time - its an alternative algorithm which was developed when $bucketAuto didnt exist.

Reply:
Will fix that test in another PR